### PR TITLE
chore: Add helm chart to repo

### PR DESCRIPTION
**Pull request by grzesuav**
Example artifact on my fork - https://github.com/grzesuav/metacontroller/pkgs/container/metacontroller-helm

### CODE REVIEW

**Comment by grzesuav** [comment]
It should produce packages like  https://github.com/grzesuav/metacontroller/pkgs/container/metacontroller-helm - I gave them distinct name `-helm` to differentiate from docker images of metacontroller

File: deploy/helm/metacontroller/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 version: v2.2.4
 appVersion: v2.2.4
```
Comment (AmitKumarDas) [inline deploy/helm/metacontroller/Chart.yaml:3]: query: Do we intend to keep both helm & app version same?
query: Are these auto magically updated every time a new release is cut (which again is automatic)?

**Review by AmitKumarDas — APPROVED** [review]

File: deploy/helm/metacontroller/Chart.yaml (hunk 1 above)
Comment (grzesuav) [reply deploy/helm/metacontroller/Chart.yaml:3]: 1. dunno, will check that, however the artifact name is determined from `version`
2. yes, they are updated by semantic-release on every release automatically

**Pull request merged**